src/schema/v2/artwork/utilities.ts

```diff
@@ -1,7 +1,7 @@
 import { parse } from "url"
 
 export const isDimensional = (value) => parseFloat(value) > 0
-export const isTinyDimensional = (value) => parseFloat(value) > 1
+export const isTinyDimensional = (value) => parseFloat(value) > 3
```

Comment:
Is the `3` coming from somewhere in particular or arbitrary? 🤔

Reply:
Chosen by stakeholders